src/botPage/view/Dialogs/Chart.js

```diff
@@ -145,10 +151,10 @@ class ChartContent extends PureComponent {
         return (
             <SmartChart
                 id={this.chartId}
+                granularity={this.state.granularity}
                 symbol={this.state.symbol}
-                isMobile={true}
+                isMobile={false}
```

Comment:
@aaron-binary ismobile should be true cause the dialogs look odd in desktop view for our small window